tests/docker/globus.debian11.Dockerfile

```diff
@@ -19,40 +20,23 @@ RUN apt-get update && apt-get install -y curl \
     python3-pyodbc
 
 #### Get and install iRODS repo ####
-RUN apt-get update && apt-get install -y wget gnupg2 lsb-release
-RUN wget -qO - https://packages.irods.org/irods-signing-key.asc | apt-key add -
-RUN echo "deb [arch=amd64] https://packages.irods.org/apt/ $(lsb_release -sc) main" | tee /etc/apt/sources.list.d/renci-irods.list
-RUN apt-get update
+RUN apt-get update && apt-get install -y wget gnupg2 lsb-release && \
+    wget -qO - https://packages.irods.org/irods-signing-key.asc | apt-key add - && \
+    echo "deb [arch=amd64] https://packages.irods.org/apt/ $(lsb_release -sc) main" | tee /etc/apt/sources.list.d/renci-irods.list && \
+    apt-get update
 
 #### Install icommands - used to set up, validate and tear down tests. ####
-RUN apt-get install -y irods-icommands
-
-#### Install irods-dev, cmake, clang  - used to build the connector ####
-RUN apt-get install -y irods-externals-cmake3.21.4-0
-RUN apt-get install -y irods-externals-clang*
-RUN apt-get install -y irods-dev
-
-#### install basic packages ####
-RUN apt-get install -y curl \
-    g++ \
-    gcc \
-    vim \
-    ftp \
-    telnet \
-    libcurl4-gnutls-dev \
-    unzip \
-    python3 \
-    python3-distro \
-    python3-psutil \
-    python3-jsonschema \
-    python3-requests \
-    python3-pip \
-    python3-pyodbc
+#### Install externals and dev package to build the connector.         ####
+RUN apt-get install -y irods-icommands \
+    irods-externals-cmake3.21.4-0 \
+    irods-externals-clang13.0.1-0 \
+    irods-externals-clang-runtime13.0.1-0 \
+    irods-dev
```

Comment:
I think it would be helpful to be explicitly clear that these are the lines which the README is instructing us to comment out. Or... if it's not, then we shuld add comments in the places indicating which lines we are supposed to comment out when installing custom iRODS packages.

Reply:
Oh, I forgot to add the comments.  Will add them.

Reply:
I rearranged the commands (so that the commented out parts will be easy) and added comments on what to  comment/uncomment.